tests/bgp/conftest.py

```diff
@@ -220,29 +220,41 @@ def _setup_interfaces_t1(mg_facts, peer_count):
             _subnets = ipaddress.ip_network(u"10.0.0.0/24").subnets(new_prefix=subnet_prefixlen)
             subnets = (_ for _ in _subnets if _ not in used_subnets)
 
+            loopback_ip = None
+            for intf in mg_facts["minigraph_lo_interfaces"]:
+                if netaddr.IPAddress(intf["addr"]).version == 4:
+                    loopback_ip = intf["addr"]
+                    break
+                if not loopback_ip:
```

Comment:
shouldn't this be outside of for loop?

Reply:
Sure will update .. this also works as there is only one loopback interface.